### DISCUSSION TITLE
Add swing timing support to the editor

### DISCUSSION THREAD

**Discussion by TheNikku** [ppy/osu#13098] [Ideas]
This is a quick video on swing in music

For your consideration, I noticed there was no tool for this in the editor for OSU or Lazer & I think many mappers would find this quite useful for groovy music.

Kind regards,
TheNikku

https://youtu.be/njFc50mC7f4

**Comment by peppy** [comment]
Got any sample beatmaps where this would have been used? Most of the time swing can be accounted for by using a 1/8 or 1/16 divisor, so examples of edge cases would be nice to have.
